apps/assets/automations/gather_facts/host/posix/main.yml

```diff
@@ -12,7 +12,12 @@
           cpu_cores: "{{ ansible_processor_cores }}"
           cpu_vcpus: "{{ ansible_processor_vcpus }}"
           memory: "{{ ansible_memtotal_mb / 1024 | round(2) }}"
-          disk_total: "{{ (ansible_mounts | map(attribute='size_total') | sum / 1024 / 1024 / 1024) | round(2) }}"
+          disk_total: |-
+            {% set ns = namespace(total=0) %}
+            {%- for name, dev in ansible_devices.items() if dev.removable == '0' and dev.host != ''  -%}
+              {%- set ns.total = ns.total + ( dev.sectors | int * dev.sectorsize | int ) -%}
+            {%- endfor -%}
+            {{- (ns.total / 1024 / 1024 / 1024) | round(2) -}}
```

Comment:
disk_total: >-
            {% set total_size = 0 %}
            {% for device_name, device in ansible_devices.items() %}
              {% if device.removable == '0' and device.host != '' %}
                {% set total_size = total_size + (device.sectors | int * device.sectorsize | int) %}
              {% endif %}
            {% endfor %}
            {{ (total_size / 1024 / 1024 / 1024) | round(2) }}